net-im/discord-bin/discord-bin-0.0.8.ebuild

```diff
@@ -0,0 +1,92 @@
+# Copyright 1999-2019 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_PN=${PN/-bin/}
+inherit eutils gnome2-utils unpacker
```

Comment:
I don't see eutils being used in here, is it?

Reply:
Removed it and it still works, so apparently not.